bitbucket_hook.go

```diff
@@ -0,0 +1,114 @@
+package git
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net"
+)
+
+// See: https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html
+var bitbucketIPBlocks = []string{
+	"131.103.20.160/27",
+	"165.254.145.0/26",
+	"104.192.143.0/24",
+}
+
+type BitbucketHook struct {}
+
+type bbPush struct {
+	Push struct {
```

Comment:
Though this is minor, it does not look like `gofmt` is executed for this source.